### PR TITLE
fixed blob column

### PR DESCRIPTION
**Pull request by boolw**

### CODE REVIEW

File: column.go [hunk 1]
```diff
@@ -312,17 +304,10 @@ type Text struct {
 	Collate string
 
 	Prefix string // tiny, medium, long
-	Blob   bool   // for binary
```
Comment (SergeyPodgornyy) [inline column.go]: What is the intention of this change? Is this related to the character set?

File: column.go [hunk 2]
```diff
@@ -312,17 +304,10 @@ type Text struct {
...
 }
 
 func (t Text) buildRow() string {
-	sql := t.Prefix
-
-	if t.Blob {
-		sql += "blob"
```
Comment (SergeyPodgornyy) [inline column.go]: If you remove it from here, then a new `major` version should be created. Can you keep it in the `Text` for a while for BC reasons, please?

File: column.go [hunk 3]
```diff
@@ -354,6 +339,44 @@ func (t Text) buildRow() string {
 	return sql
 }
 
+// Text represents long text column type represented in DB as:
```
Comment (SergeyPodgornyy) [inline column.go]: Please adjust the doc block. It describes the `Text`, but not a `Blob`

File: column.go [hunk 4]
```diff
@@ -354,6 +339,44 @@ func (t Text) buildRow() string {
...
+//			↪️ longblob NOT NULL
+
+type Blob struct {
+	Nullable bool
+	Comment  string
+
+	Prefix string // tiny, medium, long
+}
+
+func (t Blob) buildRow() string {
```
Comment (SergeyPodgornyy) [inline column.go]: nit: `func (b Blob)`

File: migrator.go [hunk 5]
```diff
@@ -232,10 +232,11 @@ func (m Migrator) createMigrationTable(db *sql.DB) error {
 			"id int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY",
 			"name varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL",
 			"batch int(11) NOT NULL",
-			"applied_at timestamp(6) NULL DEFAULT CURRENT_TIMESTAMP(6)",
+			"applied_at timestamp NULL DEFAULT CURRENT_TIMESTAMP",
```
Comment (SergeyPodgornyy) [inline migrator.go]: Any reason you removed milliseconds? This migrator is designed to be compatible with MySQL 8

File: log.go [hunk 6]
```diff
@@ -0,0 +1,16 @@
+package migrator
+
+import "log"
+
+var (
+	Debug  = false
```
Comment (SergeyPodgornyy) [inline log.go]: Global variables are not a good programming practice, so I would not agree to keep them in the codebase.

If you wanna do it the proper way, it should be set as a property of `migrator.Migrator{}`

File: log.go [hunk 7]
```diff
@@ -0,0 +1,16 @@
+package migrator
+
+import "log"
```
Comment (SergeyPodgornyy) [inline log.go]: The default `log` package is not flexible enough and I am not it is a good idea to use it here. Also, it is out of the scope of the MR title

File: column.go (hunk 1 above)
Comment (boolw) [reply column.go]: The local MySQL version is 5.5.53 and does not support this new feature. How can we find a compatible solution

File: column.go (hunk 2 above)
Comment (boolw) [reply column.go]: Ok, no problem

File: column.go (hunk 3 above)
Comment (boolw) [reply column.go]: It is mainly about compatibility with lower versions

File: column.go (hunk 4 above)
Comment (boolw) [reply column.go]: ok

File: migrator.go (hunk 5 above)
Comment (boolw) [reply migrator.go]: It is mainly about compatibility with lower versions

File: log.go (hunk 6 above)
Comment (boolw) [reply log.go]: The first idea is the same, but the place where sql is executed later is implemented by function transfer, which will be passed multiple times, and the test case is also used, and the changes are relatively large, so it is forced to use the global variable method

File: log.go (hunk 7 above)
Comment (boolw) [reply log.go]: The default log is closed. Many times when developing, you need to see the execution of sql and compare it with the error message.

File: column.go (hunk 1 above)
Comment (SergeyPodgornyy) [reply column.go]: This version is not supported even by MySQL. Then why should this project support it? 🤔 

mysql 5.5 release date

This package was designed to work with MySQL v8 and higher, therefore you'll find more features that won't be supported in your local environment. 

If it is your local version, why you can't just update it to a newer version?

File: column.go (hunk 3 above)
Comment (SergeyPodgornyy) [reply column.go]: doc block should be relevant anyway

File: migrator.go (hunk 5 above)
Comment (SergeyPodgornyy) [reply migrator.go]: Milliseconds introduced in MySQL since version 5.6 and this version is not supported by MySQL anymore

File: log.go (hunk 6 above)
Comment (SergeyPodgornyy) [reply log.go]: It is still not a good reason to use the global variable 🙂 Better explicit definition in many places instead of global scope. It does not allow you to test it and you never know how it might affect your functionality

File: log.go (hunk 7 above)
Comment (SergeyPodgornyy) [reply log.go]: I don't think we need this logger for development, especially when this feature is available for package users. 

If you need to check the output of the execution, you can use a debugger or check the output in tests.

File: column.go [hunk 8]
```diff
@@ -12,19 +12,19 @@ func (c columns) render() string {
...
 
 type column struct {
 	field      string
-	definition columnType
+	definition ColumnType
 }
 
-type columnType interface {
-	buildRow() string
+type ColumnType interface {
```
Comment (SergeyPodgornyy) [inline column.go]: Why did you change it? It works perfectly fine in private scope. Any reason it should be exported.

Also, it is out of the scope of the MR ❗

File: migrator.go (hunk 5 above)
Comment (boolw) [reply migrator.go]: ok

File: column.go (hunk 8 above)
Comment (boolw) [reply column.go]: If it is exported, developers can implement it more flexibly in outsourcing. For example, I found that when the blob type has different characteristics in different versions, it can solve similar problems externally, and the Customize queries part of the document cannot be implemented externally.
